db/migrate/20160220123313_add_open_scap_result_entities.rb

```diff
@@ -0,0 +1,23 @@
+class AddOpenScapResultEntities < ActiveRecord::Migration
+  def up
```

Comment:
This can be wrapped in a `def change` migration. E.g. https://github.com/ManageIQ/manageiq/blob/c0868248d2681c6cd6644fc8c05e11b4f255cf4f/db/migrate/20160211113430_create_cloud_databases.rb